easybuild/easyconfigs/f/Forge/Forge-21.0.2.eb

```diff
@@ -0,0 +1,44 @@
+##
+# Based on the easyconfig created by the BEAR Software team at the University of Birmingham.
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Tarball'
+
+name = 'Forge'
+version = '21.0.2'
+
+homepage = "https://www.arm.com/products/development-tools/server-and-hpc/forge"
+description = """Build reliable and optimized code for the right results on multiple Server and HPC architectures,
+ from the latest compilers and C++ 11 standards to Intel, 64-bit Arm, AMD, OpenPOWER and Nvidia GPU hardware. Arm
+ Forge combines Arm DDT, the leading debugger for time-saving high performance application debugging, Arm MAP, the
+ trusted performance profiler for invaluable optimization advice across native and Python HPC codes, and Arm
+ Performance Reports for advanced reporting capabilities."""
+
+toolchain = SYSTEM
```

Comment:
@robert-mijakovic The `Forge-21.0.2.eb` easyconfig should be removed?

Reply:
Removed! Thanks, I completely forgot about it.